content/applications/hr/employees/certifications.rst

```diff
@@ -0,0 +1,120 @@
+==============
+Certifications
+==============
+
+When jobs require specific knowledge, it is necessary to track employee certifications to ensure the
+necessary knowledge and certifications are in place.
+
+Certifications include classes, tests, professional seminars, and more. There are no restrictions in
+terms of what type of certification records can be added in Odoo.
+
+.. important::
+   To access the *Employee Certifications* report, the **Surveys** app **must** be installed.
+
+View certifications
+===================
+
+To view a full list of all employee certifications, navigate to :menuselection:`Employees app -->
+Reporting --> Certifications`.
+
+All certifications appear in a list view, grouped by employee. Each certification entry displays the
+following:
+
+- :guilabel:`Employee`: the employee's name, along with their avatar image.
+- :guilabel:`Name`: the title of the certification.
+- :guilabel:`Validity Start`: when the employee received the certification.
+- :guilabel:`Validity End`: when the certification expires.
+- :guilabel:`Certification`: the corresponding course in the **Surveys** app that was completed by
+  the employee, if applicable.
+
+The entries are also color coded. Current certifications that are still valid appear in black,
+expired certifications appear in red, and certifications that are going to expire within the next
+90 days appear in orange.
+
+.. image:: certifications/certifications.png
+   :align: center
+   :alt: The list of employee certifications.
+
+.. important::
+   **Only** certification records with the *Display Type* set to *Certification* on their
+   :ref:`certification form <certifications/form>` appear on the *Employee Certifications* report.
```

Comment:
```suggestion
   :ref:`certification form <certifications/form>` appear on the :guilabel:`Employee Certifications` report.
```
since the reader, if they are following along, will *be* on the reporting page at this moment, the use of a GUI seems more apt.